### PR TITLE
Update workflow types

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
## Description

Update workflow types:

* [x] Remove 'several'
* [x] Remove 'just'
* [x] Write shorter sentences

## Type of suggestion

* [x] Styleguide

## Product & version (if relevant)
Umbraco Forms

### CODE REVIEW

File: 13/umbraco-forms/editor/attaching-workflows/workflow-types.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ Sends the Form to a URL either as a HTTP POST or GET. The following configuratio
 
 When mapping fields, if any are selected, only those chosen will be sent in the request to the configured URL. If no fields are mapped, all will be sent.
 
-The receiving endpoint will be able to extract the form fields and values from the querystring or form collection when the method used is set to GET or POST respectively.
+The endpoint extracts form fields and values using GET for querystrings and POST for form collections.
```
Comment (alina-tincas) [inline 13/umbraco-forms/editor/attaching-workflows/workflow-types.md:160]:
```suggestion
The receiving endpoint extracts form fields and values using GET for querystrings and POST for form collections.
```

**Review by alina-tincas — CHANGES_REQUESTED** [review]
Hi @erikjanwestendorp thank you for the PR 🙌 

I have just one suggestion and can you please make the same changes to v10 and 12 as well? 😊

**Comment by erikjanwestendorp** [comment]
@alina-tincas Thanks for the review! 😄 Just updated the PR with your suggestion and updated v10 & 12.

**Review by alina-tincas — APPROVED** [review]

**Comment by alina-tincas** [comment]
Awesome, thank you for making the changes 😁 Merging this in!

**Pull request merged**